deploy/charts/trust-manager/values.yaml

```
@@ -331,6 +331,10 @@ app:
         # Additional labels to add to the ServiceMonitor.
         labels: {}
 
+  # List of target namespaces that trust-manager can write to.
+  # Empty list = cluster-wide (default behavior).
+  targetNamespaces: []
```

```suggestion
    # List of target namespaces that trust-manager can write to. By default, trust-manager can write targets in any namespace.
  # +docs:property
  # targetNamespaces: ["ns-1", "ns-2"]
```

This section is misplaced in the metrics section. I tried to push a fix to your branch, but it seems like you haven't allowed maintainers to guide you. 😒

> ERROR: Permission to orange-cloudfoundry/trust-manager.git denied to erikgb.

I suggest moving it right after the `app.trust.namespace value`.